src/components/calcite-dropdown-group/calcite-dropdown-group.tsx

```diff
@@ -71,8 +75,18 @@ export class CalciteDropdownGroup {
       </span>
     ) : null;
 
+    const dropdownSeparator =
+      this.groupPosition > 0 ? (
+        <div
+          class="dropdown-separator"
+          ref={(node) => (this.separatorEl = node)}
```

Comment:
Can you move the node storing anonymous function to a component method? This creates a new function on each render and also has the side effect of being invoked on each render.

Reply:
NP - you're referring to this pattern of setting the ref? https://github.com/Esri/calcite-components/blob/322b0adee9951b5fc4dec8d2d2ecd79d2c1447c2/src/components/calcite-panel/calcite-panel.tsx#L166

Reply:
Yep, thanks!

Reply:
Cool, this can probably be done many other places as well.

Reply:
Reminder 🕐 / breakout issue: https://github.com/Esri/calcite-components/issues/1052